<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAbrirSB</class>
 <widget class="QDialog" name="DialogAbrirSB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Inicio</string>
   </property>
   <widget class="QRadioButton" name="radioButtonChaveDireita">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chave à Direita</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMarco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Marco</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarco">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonChaveEsquerda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chave à Esquerda</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome da SB para abrir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAbrirSB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAbrirSB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
